From 2b5da254f11370cc99c24e405605da57645a001c Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Tue, 3 Jul 2012 16:59:52 +0800
Subject: ENGR00182743-1 IPU: Add non-interleaved YUV444 pixel format support

Define IPU_PIX_FMT_YUV444P macro for non-interleaved YUV444 pixel format

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 include/linux/ipu.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/ipu.h b/include/linux/ipu.h
index e0c9e90..f8178f1 100644
--- a/include/linux/ipu.h
+++ b/include/linux/ipu.h
@@ -139,6 +139,8 @@ typedef enum {
 #define IPU_PIX_FMT_YUV420P2 fourcc('Y', 'U', '1', '2')	/*!< 12 YUV 4:2:0 */
 #define IPU_PIX_FMT_YVU422P fourcc('Y', 'V', '1', '6')	/*!< 16 YVU 4:2:2 */
 #define IPU_PIX_FMT_YUV422P fourcc('4', '2', '2', 'P')	/*!< 16 YUV 4:2:2 */
+/* non-interleaved 4:4:4 */
+#define IPU_PIX_FMT_YUV444P fourcc('4', '4', '4', 'P')	/*!< 24 YUV 4:4:4 */
 /*! @} */
 #define IPU_PIX_FMT_TILED_NV12_MBALIGN	(16)
 #define TILED_NV12_FRAME_SIZE(w, h)	\
-- 
1.7.9.5

